lib/shared/ajv.js

```diff
@@ -8,27 +8,21 @@
 // Requirements
 //------------------------------------------------------------------------------
 
-const Ajv = require("ajv"),
-    metaSchema = require("ajv/lib/refs/json-schema-draft-04.json");
+const Ajv = require("ajv-draft-04");
```

Comment:
the ajv v8 was listed in dependencies, but is it required in somewhere?

Reply:
It’s a peer dependency so that updating it doesn’t require ajv-draft-04 release